src/translate.js

```diff
@@ -157,4 +163,20 @@ export default {
     return this.getTranslation(msgid, n, context, plural)
   },
 
+  /*
+   * Initialize local state for translations and configuration.
+   * Required to decouple global 'Vue'
```

Comment:
You need to be explicit somewhere in the comments and explain why you're doing this: to use vue-gettext without Vue.

Please, can you update the README too. Add documentation and at least one example describing how to use the plugin for a Node script.